src/common/components/InputField/InputField.tsx

```diff
@@ -0,0 +1,66 @@
+import { ChangeEvent } from "react";
+import classNames from "classnames/bind";
+import styles from "./InputField.module.scss";
+import { PrefixIcon, SuffixIcon, ErrorMessage, Label } from "./index";
+import Input from "./Input/Input";
+
+const cn = classNames.bind(styles);
+
+interface InputFieldProps {
+  name: string;
+  type?: "text" | "email" | "password" | "dropdown" | "search";
+  placeholder?: string;
+  size?: "sm" | "md" | "full";
+  color?: "white" | "gray";
+  isError?: true | false;
+  label?: string;
+  disabled?: true | false;
+  unit?: string;
+  prefix?: string;
+  value?: string;
+  onChange?: (e: ChangeEvent) => void;
+  border?: "solid" | "none";
+  errorMessage?: string;
+}
+
+export default function InputField({
+  name,
+  type = "text",
+  placeholder = "",
+  size = "md",
+  color = "white",
+  isError = false,
+  label,
+  disabled = false,
+  unit,
+  prefix,
+  value,
+  onChange,
+  border = "solid",
+  errorMessage,
+  ...rest
+}: InputFieldProps) {
+  const className: string = cn("default", size, color, isError && "error", border);
+
+  return (
+    <div className={styles.inputField}>
+      {label && <Label name={name} label={label} />}
+      <div className={className}>
+        {prefix && <PrefixIcon prefix={prefix} />}
```

Comment:
이렇게 조금 더 명시적으로 표현해줘도 좋을 것 같습니다~
```suggestion
        {hasPrefixIcon && <PrefixIcon prefix={prefix} />}
```

Reply:
아이콘 자체를 외부에서 주입받도록 표현해줘도 괜찮을 것 같습니다~
```ts
// 사용하는 쪽
  <InputField ... prefixIcon={<CalendarIcon />} />

// InputField 컴포넌트 내부
  {prefixIcon && prefixIcon}
```